src/components/organisms/NavBarSchedule/NavBarSchedule.tsx

```diff
@@ -108,7 +126,22 @@ export const NavBarSchedule: React.FC<NavBarScheduleProps> = ({
       }
     };
 
-    return range(SCHEDULE_SHOW_DAYS_AHEAD).map((dayIndex) => {
+    // we add 1 day so that we will still show the last day on the schedule
+    const daysTillScheduleEnded = lastDayOfSchedule
+      ? Math.max(
+          0,
+          differenceInCalendarDays(
+            addDays(lastDayOfSchedule, 1),
+            startOfToday()
+          )
+        )
+      : undefined;
+
+    const scheduleDaysToShowCount = isDefined(daysTillScheduleEnded)
+      ? Math.min(daysTillScheduleEnded, SCHEDULE_MAX_DAYS_AHEAD)
+      : SCHEDULE_MAX_DAYS_AHEAD;
+
+    return range(scheduleDaysToShowCount).map((dayIndex) => {
```

Comment:
[note] This is the logic for determining how many days are left till `lastDayOfSchedule`, and showing either the remaining days, or `SCHEDULE_MAX_DAYS_AHEAD` if there are many days to go (or if `end_utc_seconds` isn't defined)

Reply:
```
 isDefined(daysTillScheduleEnded)
      ? Math.min(daysTillScheduleEnded, SCHEDULE_MAX_DAYS_AHEAD)
      : SCHEDULE_MAX_DAYS_AHEAD;
```
Seems like an excessive logic for not setting `daysTillScheduleEnded = Infinity` since `Math.min(Infinity, X ) -> X`

Reply:
I have no idea where you’re getting ‘infinity’ from here? But I assume you’re making an incorrect assumption about the value of SCHEDULE_MAX_DAYS_AHEAS, which is set to something like 7

Reply:
If you have a suggestion for how to simplify it and maintain the existing functionality, feel free to suggest.